.github/workflows/service-build-test-upload.yml

```diff
@@ -105,7 +105,23 @@ jobs:
         run: |
           docker run --entrypoint "" ${{ env.IMAGE_NAME }} black --target-version py310 -S --check /usr/src/app
 
+      - name: Run isort
+        if: ${{ steps.changed-py-files.outputs.any_changed == 'true' && success() || failure() }}
```

Comment:
praise: I didn't know about `success() || failure()` before. Will be definitely useful in many other cases. Thanks